source/configuring-ember/specifying-url-type.md

```diff
@@ -1,4 +1,12 @@
-By default the Router uses the browser's hash to load the starting state of your
+The Ember router has four options to manage your application's URL: `auto`
+(default), `hash`, `history`, and `none`. By default, Ember CLI configures the
```

Comment:
I see there is a list of details below but a few small pieces of context may be helpful here:

> The Ember router has four options to manage your application's URL: `hash` which are anchor-based URLs, `history` which uses the HTML5 History API, `auto` and `none`.

The default behavior is the next sentence, seems needless to repeat it above. For a newb, this quick context on what hash and history are helps to explain why there is an auto behavior that is so complex.